<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>casualOnTheSpot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="configuration">
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="title">
           <string>Application Settings</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QGroupBox" name="groupBox_10">
             <property name="minimumSize">
              <size>
               <width>600</width>
               <height>0</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QGroupBox" name="groupBox_15">
                <property name="minimumSize">
                 <size>
                  <width>350</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="title">
                 <string/>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QGroupBox" name="groupBox_11">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_8">
                    <item>
                     <widget class="QLabel" name="lb_max_threads">
                      <property name="text">
                       <string>Max download workers</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="inp_max_threads"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_14">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_11">
                    <item>
                     <widget class="QLabel" name="lb_parsing_acc_sn">
                      <property name="text">
                       <string>Parsing Account SN</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="inp_parsing_acc_sn"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_12">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <item>
                     <widget class="QLabel" name="lb_download_root">
                      <property name="text">
                       <string>Download Location</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="inp_download_root"/>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btn_download_root_browse">
                      <property name="text">
                       <string>Browse</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_16">
                <property name="title">
                 <string/>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QGroupBox" name="groupBox_20">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_15">
                    <item>
                     <widget class="QLabel" name="lb_download_delay">
                      <property name="text">
                       <string>Download delay</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="inp_download_delay"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_17">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_12">
                    <item>
                     <widget class="QLabel" name="lb_max_retries">
                      <property name="text">
                       <string>Max retries</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="inp_max_retries"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_21">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="title">
                    <string/>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_10">
                    <item>
                     <widget class="QLabel" name="lm_media_format_2">
                      <property name="text">
                       <string>Max search results</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="inp_max_search_results">
                      <property name="minimum">
                       <number>1</number>
                      </property>
                      <property name="maximum">
                       <number>50</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_9">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QGroupBox" name="groupBox_19">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_14">
                 <item>
                  <widget class="QLabel" name="lm_media_format">
                   <property name="text">
                    <string>Media Format</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_media_format"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="inp_raw_download">
                <property name="text">
                 <string>Raw Media Download</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="inp_force_premium">
                <property name="text">
                 <string>Force premium</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>140</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <widget class="QPushButton" name="btn_save_config">
                   <property name="text">
                    <string>Save settings</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupbox_accounts">
          <property name="title">
           <string>Spotify Accounts</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTableWidget" name="tbl_sessions">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Username</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Account type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Action</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="group_login_form">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>90</height>
              </size>
             </property>
             <property name="title">
              <string>Add new account to pool</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="inp_login_username">
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>Spotify Username/Email</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="inp_login_password">
                <property name="text">
                 <string/>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Spotify Password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_login_add">
                <property name="text">
                 <string>Add account</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="search">
       <attribute name="title">
        <string>OnTheSpot Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="group_search">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox_7">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="title">
              <string>Search Spotify</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="inp_search_term">
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>Resham by Nepathya</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_search">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="group_dl_url">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="title">
              <string>Download by URL</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="inp_dl_url">
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>https://open.spotify.com/album/5NZQMCVRrH98PwKonS4wRy?si=895ca0d4edb945e5</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_url_download">
                <property name="text">
                 <string>Download</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="group_search_results">
          <property name="title">
           <string>Search Results</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableWidget" name="tbl_search_results">
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>By</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Content Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Action</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="group_search_actions">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="title">
              <string>Bulk Download</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="btn_search_download_all">
                <property name="text">
                 <string>All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_search_download_tracks">
                <property name="text">
                 <string>All tracks</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_search_download_albums">
                <property name="text">
                 <string>All Albums</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_search_download_artists">
                <property name="text">
                 <string>All Artists</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_search_download_playlists">
                <property name="text">
                 <string>All Playlists</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="progress">
       <attribute name="title">
        <string>Progress</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableWidget" name="tbl_dl_progress">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>MediaID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Title</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>By</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CollectionType</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Progress</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
